enhancements/console/samples.md

```diff
@@ -0,0 +1,309 @@
+
+### User Stories
+
+* As an OpenShift Serverless engineer, I want to provide additional samples that are tied to an operator version, not an OpenShift release.
+* As a cluster administrator, I want to add additional samples to the Developer Console sample catalog.
+
+### Goals
+
+1. This feature should allow other teams to provide their Developer Console samples without applying them to the sample-operator or the console-operator.
+2. Samples should be allowed to localize/translated.
+
+### Non-Goals
+
+1. We don't want to turn off the support for the (deprecated) [samples-operator](https://github.com/openshift/cluster-samples-operator) right now. So the console will not provide any samples on its own (in 4.14).
+2. A way to hide installed samples. Operators creating a sample resource should have a way to opt out of their samples.
+
+## Proposal
+
+### Workflow Description
+
+The Console Operator will provide a cluster-wide `ConsoleSample` CRD.
+
+Other operators like the OpenShift Serverless operator will install their own `ConsoleSample` CRs.
+
+The web console will automatically load all `ConsoleSample` CRs and show them in the sample catalog.
+
+Users can select one of samples and follow the import flows to create a demo application.
+
+### API Extensions
+
+The console operator automatically installs a new `ConsoleSample` CRD.
+
+### Risks and Mitigations
+
+#### The OpenShift Console is optional and could be disabled
+
+Other operators should expect that the `ConsoleSample` CRD is unavailable like the other `Console*` CRDs.
+
+### Drawbacks
+
+Another extra (but specific) custom resource just for console samples.
+
+## Design Details
+
+An "Import from Git" example of how we have it today as part of the Builder Images annotation:
+
+```yaml
+apiVersion: console.openshift.io/v1
+kind: ConsoleSample
+metadata:
+  name: java-sample
+spec:
+  title: Java
+  description: |
+    Build and run Java applications using Maven and OpenJDK.
+    Sample repository: https://github.com/jboss-openshift/openshift-quickstarts
+  icon: base64 encoded image
+  provider: Red Hat
+  badge: Serverless function
+  tags:
+  - java
+  - jboss
+  - openjdk
+  source:
+    type: GitImport
+    gitimport:
+      url: https://github.com/jboss-openshift/openshift-quickstarts
+```
+
+An "Import container image" example:
+
+```yaml
+apiVersion: console.openshift.io/v1
+kind: ConsoleSample
+metadata:
+  name: minimal-ubi-container
+spec:
+  title: Minimal UBI container
+  description: |
+    The Red Hat Universal Base Image is free to deploy on Red Hat or non-Red Hat platforms
+    and freely redistributable...
+  icon: base64 encoded image
+  provider: Red Hat
+  badge: Empty container
+  source:
+    type: ContainerImport
+    containerimport:
+      image: registry.access.redhat.com/ubi8/ubi-minimal:8.8-860
+```
+
+### Localization
+
+The `ConsoleSample` resources provide the same optional localization annotations then `ConsoleQuickStarts`.
```

Comment:
```suggestion
The `ConsoleSample` resources provide the same optional localization annotations as `ConsoleQuickStarts`.
```

Reply:
Worth linking to https://github.com/openshift/enhancements/blob/master/enhancements/console/quick-starts.md#quick-start-internationalization